tips/urls.py

```diff
@@ -7,5 +7,7 @@
     path('', views.board, name='board'),
     path('add_tip/', views.add_tip, name='add_tip'),
     path('edit_tip/<int:tip_id>/', views.edit_tip, name='edit_tip'),
-    path('delete_tip/<int:tip_id>/', views.delete_tip, name='delete_tip'),
+    path('category/<str:cats>/', views.categories_filter, name='category'),
+    path('read_more/<int:tip_id>/', views.read_more_view, name='read_more'),
+    path('delete_tip/<int:tip_id>/', views.delete_tip, name='delete_tip')
```

Comment:
I think that in house, when we inserted the house_id to the url we ran into a problem when it was int, so we changed it to str.
Am I right? @ronyyosef @ShellyGolden

Reply:
Can you please elaborate?

Reply:
What do you mean by "it is not working with int"?
Can you explain the situation couse from what I know it is working right now.
Please tell me about the functionality that isn't working and I'll try to fix it:)

Reply:
ok 👍